### DISCUSSION TITLE
Did anyone get the upscaler working in DMC5?

### DISCUSSION THREAD

**Discussion by JustABody** [praydog/REFramework#746] [General Questions]
I tried a lot of different versions of REFramework + upscaler plugin (PDPerfPlugin) and non of them work.
As soon as I add a DLSS.dll in the plugin folder, the framework window closes a few seconds after launch and the screen keeps flickering at low fps and low gpu usage... When I put the DLSS.dll in the main folder the game crashes on start up.

Any advise would be helpful.

**Comment by davidkim99b** [comment]
no luck....dev not favored

**Comment by BalisticNick** [comment]
Your game needs to be in directX12, you can change it in the settings within the game folder. Should work perfectly.
